### DISCUSSION TITLE
turning forked repository to private

### DISCUSSION THREAD

**Discussion by T1taN101** [community/community#65737] [Repositories]
### Select Topic Area

Question

### Body

Hello guys, how can I turn github forked repository from public to private?

**Comment by of1r** [comment]
You can try this tutorial: https://gist.github.com/0xjac/85097472043b697ab57ba1b1c7530274

**Comment by Binary-Ninja-007** [comment]
Good.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
